### DISCUSSION TITLE
Cannot change charge rate whilst charging

### DISCUSSION THREAD

**Discussion by ghost** [firstof9/openevse#303] [General]
With a 2019 Kona, it is not possible to change the charge rate whilst charging from HA. Changing the value causes charging to stop. Going directly via the chargers interface, there is a slider to change the value and this works perfectly whilst charging.
Is this a bug in the integration?

**Comment by firstof9** [comment]
The rate can be adjusted from HA while a charge is in progress.
Are you seeing any errors in `home-assistant.log`?

**Comment by ghost** [comment]
Thank you for responding. I just tried it again, charging simply stops as soon as I change the current from the dropdown box. I can hear the contactor disconnect immediately. There are no errors in the log file. I can start a charge from HA, and then immediately change the current from the OpenEVSE web page with no problem. Its only when I try to change the current from the HA "OpenEVSE: openevse-f214 Max Current" entity that I get a problem.

**Reply by firstof9** [reply]
Very strange, the integration uses the same commands as the OpenEVSE interface does. What wifi firmware version are you on?

**Reply by ghost** [reply]
OpenEVSE is 8.2.2.T2
The Wifi is  v5.1.0_modified
Not sure what modified means, it is a new unit I bought from the manufacturer last week.

**Reply by ghost** [reply]
Should I try downgrading?

**Reply by ghost** [reply]
I tried a few older versions of the WiFi firmware with no luck. When the problem occurs I see the following in the logbook which doesnt make much sense to me.

![image](https://github.com/firstof9/openevse/assets/63898024/de2f1ec2-f140-4c76-8dfd-451a6c631b9e)

**Reply by ghost** [reply]
I have found that direct control of the charger via MQTT allows the charge current to be set whilst charging so I am just going to go down that route.

**Reply by firstof9** [reply]
> OpenEVSE is 8.2.2.T2 The Wifi is v5.1.0_modified Not sure what modified means, it is a new unit I bought from the manufacturer last week.

I'll double check, but the `_modified` part _might_ be causing an issue for the version checks.

**Reply by firstof9** [reply]
You can use the `set_override` service call to limit the charge current.

Example limiting to 20A
```yaml
service: openevse.set_override
data:
  state: active
  auto_release: true
  max_current: 20
```
